online-shop/src/app/state/cart/cart.reducer.ts

```diff
@@ -0,0 +1,30 @@
+import { createReducer, on } from "@ngrx/store";
+import * as CartActions from './cart.actions';
+import { Status } from '../data';
+
+export interface CartState {
```

Comment:
Maybe you could move all of the states into the 'state' folder just to have the code more organized